docs/installation.md

```diff
@@ -32,6 +32,14 @@ Vue.use(VueRouter)
 
 You don't need to do this when using global script tags.
 
+## Vue CLI
+
+If you created a project using [Vue CLI](https://cli.vuejs.org/) you can add Vue Router as a [plugin](https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/cli-plugin-router). This will import the router, generate two [named routes](https://router.vuejs.org/guide/essentials/named-routes.html), and overwrite `App.vue` to include two router links.
```

Comment:
```suggestion
If you have a project using [Vue CLI](https://cli.vuejs.org/) you can add Vue Router as a plugin. You can let the CLI generate the code above for you as well as two sample routes. **It will also overwrite your `App.vue`** so make sure to backup up the file before running the following command inside your project:
```